front_end/src/components/landing_page/Landing.css

```diff
@@ -0,0 +1,84 @@
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+    top: 50px;
+  }
+  to {
+    opacity: 1;
+    top: 0px;
+  }
+}
+
+@keyframes fadeOut {
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
+}
+
+.fadeIn {
+  position: relative;
+  animation: fadeIn;
+  animation-duration: 1.5s;
+}
+
+.fadeOut {
+  animation: fadeOut;
+  animation-duration: 1.5s;
+}
+
+@keyframes moveUp {
+  from {
+    bottom: 0px;
+  }
+  to {
+    bottom: 7px;
+  }
+}
+
+@keyframes moveDown {
+  from {
+    bottom: 7px;
+  }
+  to {
+    bottom: 0px;
+  }
+}
+
+.mouseHover:hover {
+  position: relative;
+  animation: moveUp;
+  animation-duration: 0.25s;
+  animation-fill-mode: forwards;
+}
+
+.mouseHover {
+  position: relative;
+  animation: moveDown;
+  animation-duration: 0.25s;
+  animation-fill-mode: forwards;
+}
+
+.background-slideshow {
```

Comment:
do we need this if we already have the matrialUI "imgSlideShow" styles?

it's on line line 47 of Landing.js

Reply:
Removed.